ui/src/app/util/properties.util.ts

```diff
@@ -118,65 +117,61 @@ function parsePropertiesV1Permissive (properties: unknown, errorCallback: (err:
     return {}
   }
   return properties.reduce((prev: PackagePropertiesV2, cur: unknown, idx: number) => {
-    if(isSchemaV1(cur)) {
-      prev[cur.name] = {
+    
+      const result = matchPropertiesV1.enumParsed(cur);
+      if ('value' in result) {
+        const value = result.value
+      prev[value.name] = {
         type: 'string',
-        value: cur.value,
-        description: cur.description,
-        copyable: cur.copyable,
-        qr: cur.qr,
+        value: value.value,
+        description: value.description,
+        copyable: value.copyable,
+        qr: value.qr,
         masked: false,
       }
     }
-    else if (schemaV1Compiled.errors) {
-      for (let err of schemaV1Compiled.errors) {
-        errorCallback(new Error(`/data/${idx}${err.dataPath}: ${err.message}`))
-        if (err.dataPath) {
-          applyOperation(cur, { op: 'replace', path: err.dataPath, value: undefined })
-        }
-      }
-      if (!schemaV1CompiledWithDefaults(cur)) {
-        for (let err of schemaV1CompiledWithDefaults.errors) {
-          errorCallback(new Error(`/data/${idx}${err.dataPath}: ${err.message}`))
-        }
-        return prev
+    else {
+      const error = result.error;
+      const message = Parser.validatorErrorAsString(error)
+      let dataPath = error.keys.map(x => JSON.parse(x)).join('/')
+      errorCallback(new Error(`/data/${idx}: ${message}`))
+      if (dataPath) {
+        applyOperation(cur, { op: 'replace', path: dataPath, value: undefined })
```

Comment:
get clarification on what lines 133-139 are doing in the first place to ensure proper transformation

@MattDHill @DrewAns can you explain the purpose of lines 131-137 in the original file?

Reply:
I'm not sure if the errorCallback is right at all, or the applyOperation